docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md

```diff
@@ -355,7 +464,37 @@ yOffset = 100
 webdriver.ActionChains(driver).move_by_offset(xOffset,yOffset).perform()
   {{< / code-panel >}}
   {{< code-panel language="csharp" >}}
-// Please raise a PR
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Interactions;
+
+namespace SeleniumApp
+{
+    public class MoveByOffset
+    {
+        public static void Main(string[] agrs)
+        {
+            IWebDriver driver = new ChromeDriver();
+            try
+            {
+                // Navigate to Url
+                driver.Navigate().GoToUrl("https://google.com");
+                // Store 'google search' button web element
+                IWebElement searchBtn = driver.FindElement(By.LinkText("Gmail"));
```

Comment:
can you please change variable name to 'gmailLink' and also can you update the comment.